pages/services/upload.vue

```diff
@@ -24,8 +24,9 @@ useSeo(
 );
 
 const user = (await getCurrentUser()) as User;
-// Fetch current session ID for navigation purposes.
-const uploadSessionId = useCookie("uploadSessionId");
+const idToken = await user.getIdToken();
+// Create a new upload session ID. Required for batch upload.
+const uploadSessionId = useSessionId().createUploadSessionId();
```

Comment:
@coderabbitai create an issue to review handling of uploadSessionId when user logs out or when tries to access the dashboard on a newly created session id

Reply:
@coderabbitai try again